src/ui/osx/TogglDesktop/Feature/TimeEntryEditor/EditorPopover.swift

```diff
@@ -27,6 +27,8 @@ final class EditorPopover: NoVibrantPopoverView {
     @objc func prepareViewController() {
         let editor = EditorViewController(nibName: NSNib.Name("EditorViewController"), bundle: nil)
         editor.view.appearance = appearance
+        let size = DesktopLibraryBridge.shared().getEditorWindowSize()
+        editor.view.frame.size = size
```

Comment:
Could we also use `contentSize` here?

Reply:
Actually, we don't need to override the `contentSize`, since when we set the `contentViewController` in line 32, it will automatically override `contentSize`of Popover  in

https://github.com/toggl/toggldesktop/blob/29d6aee704e140c5b702c12ec4cd4ee508eacc09/src/ui/osx/TogglDesktop/Feature/TimeEntryEditor/ResizablePopover.swift#L37

and 

https://github.com/toggl/toggldesktop/blob/29d6aee704e140c5b702c12ec4cd4ee508eacc09/src/ui/osx/TogglDesktop/Feature/TimeEntryEditor/ResizablePopover.swift#L180

Reply:
Ah, got it, thanks 👍